automation/vars/main.yml

```diff
@@ -174,6 +174,12 @@ consul_services:
 #      - { http: "http://{{ inventory_hostname }}:{{ patroni_restapi_port }}/async?lag={{ patroni_maximum_lag_on_replica }}", interval: "2s" }
 #      - { args: ["systemctl", "status", "pgbouncer"], interval: "5s" }
 
+# TLS certificate (for PostgreSQL & PGBouncer)
+tls_cert_generate: true
+tls_cert_valid_days: 3650
```

Comment:
According to the documentation: The minimum certificate lifetime is 90 days, and maximum is three years.